src/cargo/core/resolver/mod.rs

```diff
@@ -483,7 +478,7 @@ impl<T> RcVecIter<T> {
 impl<T> Clone for RcVecIter<T> {
     fn clone(&self) -> RcVecIter<T> {
         RcVecIter {
-            vec: self.vec.clone(),
+            vec: Rc::clone(&self.vec),
```

Comment:
Let's use `.clone()`

Reply:
FWIW this style is now considered "standard" overall, apparently

Reply:
crates.io adapted it, https://github.com/rust-lang/crates.io/pull/1082

Reply:
I'm not sure I agree this is standard yet (e.g. the libs team afaik hasn't discussed this much), and I personally would prefer to not switch to this.